build/wd_ts_bundle.bzl

```diff
@@ -80,6 +80,7 @@ def wd_ts_bundle_capnp(
     if lint:
         # todo: lint js_srcs too, not just ts_srcs
         eslint_bin.eslint_test(
+            size = "enormous",
```

Comment:
Nit: I haven't encountered the @eslint job being overly slow before. Not to bikeshed too much but large should be sufficient right?

Reply:
previously we weren't using typescript-eslint correctly. type-aware linting wasn't running, hence, we didn't encounter this before. this will increase with the amount of js+ts code we write.

Reply:
That makes sense to me. My point is that large (60s) should be sufficient and we won't need enormous (240s) – at present the two lint jobs appear to take 4.7s and 8.8s, respectively based on CI. I think the intention behind these categories is in part to avoid having enormous tests where possible to avoid a CI slowdown, this fits well within a 60s window.

Reply:
Ah I see. Sounds good. I'll change it to `large`. Thanks for the review & explanation.